mypyc/analysis/dataflow.py

```diff
@@ -128,100 +128,100 @@ def __str__(self) -> str:
         return 'before: %s\nafter: %s\n' % (self.before, self.after)
 
 
-GenAndKill = Tuple[Set[Value], Set[Value]]
```

Comment:
Why not `GenAndKill = Tuple[Set[T], Set[T]]`?

Reply:
A good idea! Will do.